### PR TITLE
Creating a beeware application

### LINKED ISSUE #5551: Creating an application using BeeWare
### Topic Suggestion
Creating an application using BeeWare

We ask that you please be patient as our team works through approving and publishing all articles/tutorials in a timely manner.

Allow 1-3 days for a topic to be reviewed and/or approved - allow 3-7 days for an articles to be reviewed and/or published (subject to vary depending on _volume and/or backlog of articles_).

Be sure to visit our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for tools, resources, and example articles that will help you propose and write a successful article.

**Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**We tend to stray away or tend not to publish reviews/comparisons of commercial product offerings.**

### Writing sample(s): 
Include any links or writing samples - to help our team better gauge your writing quality.

#### Proposed title of article
Creating an application using Beeware.

Use keyword research to improve your article's SEO.

#### Proposed article introduction
Beeware is a python GUI framework that you can use to create native applications that can run across all major platforms. It can serve as an alternative to other tools such as kivy. In this article we will be building an application using beeware.

For example, if the article is based on machine learning, use the following: [Machine learning] Introduction to Machine Learning.
If the article is based on developing an Android application, use the following: [Android] Developing Apps using Android.

ONLY viable topics to pick from:
- API
- Artificial Intelligence
- Blockchain
- Containers
- Edge Computing
- Languages
- Machine Learning
- Networking
- Node.js
- Security

#### Key takeaways 
1.Learn how to create a cross-platform application using the BeeWare library.
2. Improve your python skills especially in relation to object oriented programming.

#### Article quality
I intend to create a simple calculator using Beeware and maybe ship it for android. The article will introduce the readers to the BeeWare library and appreciate its way of building apps.

#### References
Please list links to any published content/research that you intend to use to support/guide this article.

### Conclusion
Finally, remove the Pre-Submission advice section and all our blockquoted notes as you fill in the form before you submit.
We look forwarding to reviewing your topic suggestion.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by JohnKiguru**
## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #5551 

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

File: content/articles/creating-an-application-using-beeware/index.md
Comment (jerimkaura) [inline content/articles/creating-an-application-using-beeware/index.md:67]:
```suggestion
We will now modify the `app.py` file step by step in order to create our final calculator application. We will start by defining all the necessary box components required. Modify the `app.py` file as follows:
```

File: content/articles/creating-an-application-using-beeware/index.md
Comment (jerimkaura) [inline content/articles/creating-an-application-using-beeware/index.md:105]:
```suggestion
We begin by creating six boxes and one main box that will have all the six boxes inside. We have the main box as a column box, meaning it will have all width by default unless defined and height will expand according to the content within the box.
```

**Review by jerimkaura — COMMENTED** [review]
Hey @JohnKiguru 

Would you please take a look at the suggestions and work on them before we proceed?

File: content/articles/creating-an-application-using-beeware/index.md
Comment (jerimkaura) [inline content/articles/creating-an-application-using-beeware/index.md:243]: Function names and variables should be enclosed in backticks. Do that across the entire article.

**Review by jerimkaura — COMMENTED** [review]
Congratulations @JohnKiguru 

Check on the comments then work on them. You have a great article.

**Comment by JohnKiguru** [comment]
Done...

File: content/articles/creating-an-application-using-beeware/index.md
Comment (jerimkaura) [reply content/articles/creating-an-application-using-beeware/index.md]: > Tell the reader to first create the folder. EXplain the required details clearly

You did not work on this. Please ensure you work on all the suggestions. Don't be in a hurry just go over the article with attention to smaller details to ensure it is error-free. Remember we encourage our authors only to publish well-refined articles.

**Review by jerimkaura — COMMENTED** [review]
Proceeding to tech review

**Review by jerimkaura — COMMENTED** [review]
Take a look at the suggetsion and work on it.

**Review by jerimkaura — APPROVED** [review]
Thank you for submitting such a great article. Congrats :+1:

**Review by jerimkaura — CHANGES_REQUESTED** [review]
Hey, @JohnKiguru  would you please add a hero image to your article, please?

**Comment by JohnKiguru** [comment]
> Thank you for submitting such a great article. Congrats +1

Thanks

**Comment by jerimkaura** [comment]
Add the hero image

**Comment by JohnKiguru** [comment]
Done...

**Review by jerimkaura — CHANGES_REQUESTED** [review]
Hey @JohnKiguru  

Thanks for adding the hero image. We are almost done here. Just one more thing,  resize the image to about 600 * 400 pixels. You could use online image editors like [canva](canva.com) or [pixlr](https://pixlr.com/editor/) for the same. Ping me after you are done with that.

**Comment by JohnKiguru** [comment]
Done...

**Comment by WanjaMIKE** [comment]
Plagiarism check done

**Review by jerimkaura — APPROVED** [review]
Great article @JohnKiguru  :rocket: 

Thank you for your patience and cooperation as we went over the review.  Your cooperation gives us an easy time with the review. Make sure you go over the changes any reviewer or I made(will make) to ensure your future articles are polished even better. 

Let us wait for the final review. Great work gentleman :handshake:

**Comment by JohnKiguru** [comment]
Thanks

**Comment by LinusMuema** [comment]
Hi @JohnKiguru 

Great Job on the article. Check on the changes made during review to improve in future articles.

cc: @jerimkaura

**Pull request merged**
